trade-finance-manager-api/src/v1/deals/routes.js

```diff
@@ -76,6 +77,10 @@ dealsAuthRouter
   .get(validation.dealIdValidation, handleExpressValidatorResult, dealController.getDeal)
   .put(validation.dealIdValidation, handleExpressValidatorResult, dealController.updateDeal);
 
+dealsAuthRouter
+  .route('/deals/:dealId/cancellation')
+  .put(validation.dealIdValidation, handleExpressValidatorResult, dealCancellationController.updateDealCancellation);
```

Comment:
In a future PR are we going to add middleware here to check that it's a user that can cancel a deal (PIM group/user?)